lib/web/ui/app.go

```diff
@@ -47,6 +47,16 @@ type App struct {
 	AWSRoles []aws.Role `json:"awsRoles,omitempty"`
 	// FriendlyName is a friendly name for the app.
 	FriendlyName string `json:"friendlyName,omitempty"`
+	// UserGroups is a list of associated user groups.
+	UserGroups []UserGroupAndDescription `json:"user_groups,omitempty"`
```

Comment:
Should the JSON property be camelCase here for consistency with the rest of the struct?

Reply:
Ahhh, good point. Will do!